lib/line/bot/request.rb

```diff
@@ -42,8 +44,18 @@ def payload
 
       # @return [Hash]
       def header
+        content_type =
+          if file.is_a? File
+            case file.path
+            when /\.png$/ then 'image/png'
+            when /\.jpe?g$/ then 'image/jpeg'
+            end
```

Comment:
i found 2 problem in this code.

1. if file path is '.gif' it will be content type = nil.
2. This regexp is case sensitive. If file path is '.PNG' it will be content type = nil